cwd/server.js

```diff
@@ -35,5 +35,9 @@ router.use(express.static(__dirname));
 
 router.get('/editor', (req, res) => {
   console.log('Received GET request at /editor');
-  res.sendStatus(200);
+  res.send("okay");
+});
+
+router.get('/errorgauge', (req, res) => {
```

Comment:
I wonder if it may be more efficient to just link to the `/images` path directly later on?

Reply:
Yep meant to eliminate all this